<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniSQLClass</class>
 <widget class="QWidget" name="MiniSQLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniSQL</string>
  </property>
  <widget class="QTextEdit" name="miniCmd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>741</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SQLlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入SQL语句:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SQLlineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Scriptlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件执行SQL脚本:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filePathBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="fileSelectButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="execButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>执行脚本</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MiniSQL.qrc"/>
 </resources>
 <connections/>
</ui>
